ovos_PHAL_plugin_mk1/arduino.py

```diff
@@ -66,6 +66,7 @@ def process(self, data):
             # This happens in response to the "system.version" message
             # sent during the construction of Enclosure()
             self.bus.emit(Message("enclosure.started"))
+            self.bus.emit(Message("version"))
```

Comment:
why was this added? if we are introducing a new message make it mk1 specific, version is too generic, also no message.data?

Reply:
It was for testing and I forgot to take it back out.  But, `system.version` does not work, only `version` works, and only if the newest firmware is loaded to the arduino.  Otherwise it just returns an echo.  Eventually that is how I will do validation and auto upgrade of firmware.

Reply:
but that would be writen to the serial, not in messagebus?